annotations/static/annotations/js/bar_d3.js

```diff
@@ -0,0 +1,62 @@
+var margin = {top: 20, right: 20, bottom: 30, left: 40},
+    width = 1050 - margin.left - margin.right,
```

Comment:
Let's make this responsive. 

![image](https://cloud.githubusercontent.com/assets/3451594/14258831/31ffb3a8-fa72-11e5-86fa-d66f024f02dc.png)

Consider using jQuery to get the [width](http://api.jquery.com/width/) of the parent div. You can watch ``$(window).resize`` to update accordingly, and wrap anything that depends on ``width`` in a function so that you can redraw.